packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormat.ts

```diff
@@ -1,25 +1,48 @@
+import type { JsonObject } from '../../../../types'
+import type { SingleOrArray } from '../../../../utils'
+import type { IssuerOptions } from '../../../vc/models/credential/Issuer'
 import type { W3cCredential } from '../../../vc/models/credential/W3cCredential'
 import type { CredentialFormat } from '../CredentialFormat'
 import type { JsonLdOptionsRFC0593 } from './JsonLdOptionsRFC0593'
 
-export interface JsonLdAcceptRequestOptions extends SignCredentialOptionsRFC0593 {
-  verificationMethod?: string
+export interface JsonCredential {
+  '@context': Array<string> | JsonObject
+  id?: string
+  type: Array<string>
+  issuer: string | IssuerOptions
+  issuanceDate: string
+  expirationDate?: string
+  credentialSubject: SingleOrArray<JsonObject>
```

Comment:
this will allow for extra properties to be defined on the credential

```suggestion
  credentialSubject: SingleOrArray<JsonObject>
  [key: string]: unknown
```